shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -402,12 +403,30 @@ - (BOOL)flt_hasFirstResponderInViewHierarchySubtree {
                                CGRectGetWidth(flutter_view.bounds),
                                CGRectGetHeight(flutter_view.bounds))] autorelease];
 
+  NSMutableArray* blurRadii =
+      [[NSMutableArray alloc] init];  // TODO EMILY: is autorelease the best option?
+  NSNumber* blurRadius;
+
+  //    int numFilters = 0;
```

Comment:
remove.

Reply:
The commented line of code is used for visual simulator tests. I commented it and I'll remove it before landing the final PR.